docker-compose-rails/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:3.0.1
+
+RUN apt update -qq && apt install -y nodejs npm postgresql-client
+RUN npm install --global yarn
```

Comment:
Since this is meant to be a tutorial of sorts, would you be up for adding some comments for what the various lines do? I went this direction in the [php example](https://github.com/teamairship/docker-examples/blob/main/example-php-apache/Dockerfile) - what do you think? I know it's verbose but for someone totally new to Docker, it would be helpful to have a line-by-line explanation.